packages/application-generic/src/services/readiness/readiness.service.ts

```diff
@@ -85,6 +106,12 @@ export class ReadinessService {
           throw error;
         }
       }
+    } else {
+      const error = new Error('Queues are not enabled');
+
+      Logger.error(error, 'Queues are not enabled', LOG_CONTEXT);
+
+      throw error;
```

Comment:
In case the service is unhealthy we should throw exceptions so we won't stick with zombie instances.